hpctestlib/system/ssh/host_keys.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import time
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.typecheck as typ
+
+
+@rfm.simple_test
+class host_keys_check(rfm.RunOnlyRegressionTest):
+    '''host keys age check
+
+    The host keys should be renewed regularly.
```

Comment:
I would skip this sentence, as this is a policy. I would simply state what the test does: checks if the list of SSH keys has been updated recently.